.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+default_install_hook_types: [pre-commit]
+
+repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.263"
+    hooks:
+      - id: ruff
+        stages: [commit]
```

Comment:
missing newline. might as well add the 'end of file fixer' hook?

Reply:
when it comes to non-python files I prefer to use prettier which include all the pre-commit linter hooks in one. I think it could deserve it's own PR as it leads to many changes. let me edit the file